ksqldb-test-util/src/test/java/io/confluent/ksql/test/util/ZooKeeperEmbeddedTest.java

```diff
@@ -34,14 +34,6 @@
   private static final long CONNECT_TIMEOUT_MS = (int) EmbeddedSingleNodeKafkaCluster
       .ZK_CONNECT_TIMEOUT.toMillis();
 
-  /**
-   * Test is only valid if Jetty is on the class path:
-   */
-  @SuppressWarnings("unused")
-  private final org.eclipse.jetty.server.Connector ensureClassOnClassPath = null;
```

Comment:
Out of curiosity, do you understand what these were originally for?

Reply:
Don't know really but I think it's trying to verify that the old API server is on the classpath.